ddp-workspace/projects/ddp-sdk/src/lib/services/serviceAgents/subjectInvitationServiceAgent.service.ts

```diff
@@ -21,22 +21,33 @@ export class SubjectInvitationServiceAgent extends AdminServiceAgent<any> {
     }
 
     public updateInvitationDetails(invitationId: string, notes: string): Observable<any | null> {
-        return this.postObservable(`/studies/${this.configuration.studyGuid}/invitation-details`, { invitationId, notes });
+        return this.postObservable(`/admin/studies/${this.configuration.studyGuid}/invitation-details`, { invitationId, notes });
     }
 
     public lookupInvitation(invitationId: string): Observable<StudySubject | null> {
-        return this.postObservable(`/studies/${this.configuration.studyGuid}/invitation-lookup`, { invitationId }).pipe(
+        return this.postObservable(`/admin/studies/${this.configuration.studyGuid}/invitation-lookup`, { invitationId }).pipe(
             map(response => response ? response.body : null)
         );
     }
 
     public createStudyParticipant(invitationId: string): Observable<UserGuid | null> {
-        return this.postObservable(`/studies/${this.configuration.studyGuid}/participants`, { invitationId }).pipe(
+        return this.postObservable(`/admin/studies/${this.configuration.studyGuid}/participants`, { invitationId }).pipe(
             map(response => response ? response.body : null)
         );
     }
 
-    public createUserLoginAccount(userGuid: string, email: string): Observable<any | null> {
-        return this.postObservable(`/studies/${this.configuration.studyGuid}/user/${userGuid}/login-account`, { email });
+    public createUserLoginAccount(userGuid: string, email: string): Observable<string | null> {
+        return this.postObservable(`/admin/studies/${this.configuration.studyGuid}/user/${userGuid}/login-account`, { email });
+    }
+
+    public verifyZipCode(invitationId: string, zipCode: string): Observable<any> {
+        const payload: InvitationCheckPayload = {
+            invitationId,
+            qualificationDetails: {
+                zipCode
+            },
+            auth0ClientId: this.configuration.auth0ClientId
+        };
+        return this.postObservable(`/studies/${this.configuration.studyGuid}/invitation-check`, payload);
```

Comment:
Now, if we want to check the invitation and zip code, we can provide recaptcha token or auth token. Since CRC is logged in we will provide an auth token, not to ask them to submit recaptcha.